site/includes/sign-on-off.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="row">
+{{#is signed "off"}}
```

Comment:
The link changes, start with https://clegc-gckey.gc.ca/ and click forward.

I'm not sure precedent and modern design back the exclusive "in content" argument.  I grant there are some design flows where the site requires credentials to function, where you funnel users to a registration page.  The CKAN registry doesn't work like that, it's more like twitter or google.

CKAN allows for near full functionality as a guest and as such they don't need to be clobbered over the head with login information.  They should be able to seamlessly use the site without realizing you can login till they reach a point where it's required.  In those models it's nearly convention that the optional login information be top right.

Google:
![image](https://cloud.githubusercontent.com/assets/1154737/9748737/e046d53a-5654-11e5-919e-7e2401234d05.png)

Twitter:
![image](https://cloud.githubusercontent.com/assets/1154737/9748749/0a5b88e8-5655-11e5-8f39-84830b31dfd6.png)

I'd even prefer it further top then right.  Meaning if you take the canada.ca look I'd hope for top white bar and this intranet look top grey.  To the left side of the page or just left of the language.

![image](https://cloud.githubusercontent.com/assets/1154737/9748847/d916599c-5655-11e5-8df3-62af4f8dbbf8.png)

![image](https://cloud.githubusercontent.com/assets/1154737/9748887/0f01ed1e-5656-11e5-9489-b862f4d767cd.png)

What Laurent proposes may be the best compromise considering mobile, accessibility and other considerations.  His example could be better with smaller buttons but the least disruption to the page is to the left of the language and would be my preference.

One thing that is sure, we need this optional credential functionality as Laurent proposes.  How it's implemented seems to be the only question.